examples/list-tasks.ts

```diff
@@ -0,0 +1,40 @@
+// import cw, { morgen } from "morgen-cw-sdk";
+import cw, { sandbox } from "../src";
+
+const { morgen, log } = sandbox.util;
+
+/**
+ * This example will list tasks for a provided type of task integration.
+ */
+const wf = cw.workflow(
+  {
+    name: "SDK Example: List all tasks",
+  },
+  async function run() {
+    const integs = await morgen().accounts.listAccountsV3();
+
+    const serviceName = "todoist"; // One of googleTasks, microsoftToDo, microsoftOutlook, todoist
+
+    const accountId = integs.data?.accounts?.find(
+      (integ) => integ.integrationId === serviceName
+    )?.providerId;
+
+    if (!accountId) {
+      throw new Error("User has not connected this type of account");
+    }
+
+    const taskResp = await morgen().tasks.listTasksV2({
+      accountId,
+      showCompleted: true,
```

Comment:
The APIs never return completed third-party tasks. This is by design because 1) some providers (eg. Todoist) don't support retrieving completed tasks, and 2) there might be too many completed tasks so without a pagination mechanism this would not work.

Reply:
Ah. I might have told an important potential custom workflow that we will fix this but I can let him know that this won't be possible I guess? Kind of a shame when it's possible on Google directly, it might push them to just go direct to Google via their API 🤷 and we can surely just put a sensible limit to pass to the Google API, and allow control of that too?

Reply:
What is the use case for this?

Reply:
Yep, great question - I asked him more about that